dask/base.py

```diff
@@ -1229,6 +1229,10 @@ def normalize_categorical_dtype(dtype):
     def normalize_period_dtype(dtype):
         return normalize_token(dtype.name)
 
+    @normalize_token.register(pd.NA)
+    def normalize_na(dtype):
+        return pd.NA
```

Comment:
Hmm. Cases like this certainly make the behavior of `normalize` feel a bit strange to me. I suppose we just noee **something** to avoid the general `obj` fallback.

Reply:
Yep and I didn't want to put it into the identity block above